src/github.ts

```diff
@@ -0,0 +1,151 @@
+import * as core from '@actions/core'
+import {Octokit} from '@octokit/rest'
+
+const token: string = core.getInput('token', {required: true})
+const octokit = new Octokit({auth: token})
+
+export async function forkRepo(
+  owner: string,
+  repo: string,
+  org?: string
+): Promise<void> {
+  try {
+    const res = await octokit.request('POST /repos/{owner}/{repo}/forks', {
+      owner,
+      repo,
+      organization: org ? org : ''
+    })
+    // Forks requests are still 'Accepted' (202) if the repository already exists at the specified location
+    // However, repositories with the same name but a different source are auto-incremented (eg. my-forked-repo-1)
+    if (res.status === 202) {
```

Comment:
Would it make sense to replace these ints with named constants? `HTTP_OK`, `HTTP_ACCEPTED`, `HTTP_FORBIDDEN`, etc for the sake of readability? Not something worth delaying over, but possibly a future tweak.

Reply:
I like that as a future enhancement, for sure. Given that there's also some Jest testing improvements that could be made, feels like reasonable fodder for a `v0.0.2` release. Thanks for the suggestion!